### PR TITLE
Add batman paper

### PR DESCRIPTION
**Pull request by MagnusKoudahl**

### CODE REVIEW

**Comment by bertdv** [comment]
We are required by Dutch law to publish open access. IEEE ICASSP does not provide open access (which sucks big time after paying 800 EUR for ICASSP registration!). So let's put the pdf on our own website so all can access it.

File: content/publication/batman.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+abstract_short = ""
+date = "2020-05-07T18:30:00+02:00"
+image = ""
+image_preview = ""
+math = false
+publication = "ICASSP 2020 - 2020 IEEE International Conference on Acoustics, Speech and Signal Processing (ICASSP)"
+publication_short = ""
```
Comment (albertpod) [inline content/publication/batman.md:9]: Thanks for PR @MagnusKoudahl 
Please erase "ICASSP 2020" from publication parameter and add it to publication short.

Comment (MagnusKoudahl) [reply content/publication/batman.md]: :+1:

**Comment by ivan-bocharov** [comment]
Thanks for the PR, @MagnusKoudahl

**Pull request merged**